css/src/utils/flex.css

```diff
@@ -169,3 +169,183 @@
 .order-last {
   order: 6 !important;
 }
+
+/* Gap utilities */
+.gap-0 {
+  gap: 0 !important;
+}
+
+.gap-1 {
+  gap: var(--spacing-2-5) !important;
+}
+
+.gap-2 {
+  gap: var(--spacing-05) !important;
+}
+
+.gap-3 {
+  gap: var(--spacing-10) !important;
+}
+
+.gap-4 {
+  gap: var(--spacing-20) !important;
+}
+
+.gap-5 {
+  gap: var(--spacing-60) !important;
+}
```

Comment:
Please follow our predefined guidelines for defining the tokens and classnames. The correct classnames for gap would be using the formula  `(x / 4) * 100`:
- **gap-2-5**:  1px
- **gap-05**: 2px
- **gap-7-5**: 3px
- **gap-10**: 4px
- **gap-15**: 6px
- **gap-20**: 8px
- **gap-30**: 12px